### DISCUSSION TITLE
Discord server invite link updated

### DISCUSSION THREAD

**Discussion by cyrite** [cyritegamestudios/trust#4] [Announcements]
Hey folks, the link to the Discord server has been updated. You can join [here](https://discord.gg/CfPxDy759J).

**Comment by siune-one** [comment]
Hello, anyone have an updated discord server link? This one seems to be invalid now.

**Reply by cyrite** [reply]
@siune-one the one above should work, but if you are having issues you can try this one: https://discord.gg/zxKjJBUE
